_posts/2017-12-10-trl-mesh-test.md

```diff
@@ -0,0 +1,13 @@
+---
+layout: post
+title: "Scale Mesh Test"
+slug: trl-mesh-test
+text: 'Bring your nodes, bring your computers, bring your phones,  and help us set up and test a mesh at scale'
+location: Toronto Refrence Library, Second Floor, 789 Yonge Street  
+locationLink: http://osm.org/go/ZX6B8BWGi?way=28156063
+date: 2017-12-10
+startTime: '13:00'
+endTime: '16:00'
+---
+
+Join us at {{page.startTime | date:"%-I:%M%P "}} on {{ page.date | date: '%B %d, %Y' }} to help deploy and test the first at scale TOMesh mesh. Help us discover its capabilities, downfalls and maybe learn a thing or two you didn't know before!  Our end goal with these tests is to learn enough to deploy a TOMesh at 2018's Maker Festival which happens in the Toronto Reference Library in the summer. Meet us at [{{ page.location }}]({{ page.locationLink }}) and help out.
```

Comment:
Might want to re-ask people to BYOD in the body as it doesn't appear on the event page.
Also do you want to specify like "second floor by the XXX (elevators? pods?)